packages/xo-web/src/xo-app/sr/tab-advanced.js

```diff
@@ -14,20 +14,25 @@ import { flowRight, isEmpty, keys, sum, values } from 'lodash'
 
 const COLUMNS = [
   {
+    itemRenderer: _ => <span>{_.name_label}</span>,
     name: _('srUnhealthyVdiNameLabel'),
-    itemRenderer: vdi => <span>{vdi.name_label}</span>,
-    sortCriteria: vdi => vdi.name_label,
+    sortCriteria: 'name_label',
   },
   {
-    name: _('srUnhealthyVdiSize'),
     itemRenderer: vdi => formatSize(vdi.size),
-    sortCriteria: vdi => vdi.size,
+    name: _('srUnhealthyVdiSize'),
+    sortCriteria: 'size',
   },
   {
-    name: _('srUnhealthyVdiDepth'),
     itemRenderer: (vdi, chains) => chains[vdi.uuid],
+    name: _('srUnhealthyVdiDepth'),
     sortCriteria: (vdi, chains) => chains[vdi.uuid],
   },
+  {
+    itemRenderer: _ => <Copiable>{_.uuid}</Copiable>,
```

Comment:
By default `Copiable` is a `span`. But a span doesn't use the whole space. Therefore, when you hover over the UUID and try to click on the copy button, it disappears. Adding `tagName='div'` to `Copiable` should fix it.